langgraph/checkpoint/sqlite.py

```diff
@@ -382,3 +437,46 @@ def put(
                 "thread_ts": checkpoint["ts"],
             }
         }
+
+
+def search_where(metadata_query: CheckpointMetadata, predicates: List[str] = []) -> str:
```

Comment:
Update implementation so that query values are parameterized instead of hardcoded into query.